packages/ketcher-polymer-editor-react/src/components/modal/Open/TextEditor/TextEditor.tsx

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import styled from '@emotion/styled'
+
+export type TextEditorProps = {
+  structStr: string
+  inputHandler: (str: string) => void
+}
+
+const StyledTextarea = styled.textarea`
+  width: 405px;
+  min-width: 32em;
+  min-height: 23em;
+  overflow: auto;
+  white-space: pre;
+  resize: both;
+`
+
+export const TextEditor = ({ structStr, inputHandler }: TextEditorProps) => {
```

Comment:
Maybe we should have a shared styled textarea component, same one we create for Save modal? https://github.com/epam/ketcher/pull/1246/files#diff-29ff483d977e475e9b6e4578a4b165776d0c3f268c75f2d8a5cc6140b78b0b32